src/components/OpenWith/OpenWith.js

```diff
@@ -208,22 +217,85 @@ class OpenWith extends PureComponent<Props, State> {
     }
 
     /**
-     * Renders the Open With element
+     * Click handler when an integration is clicked
+     *
+     * @private
+     * @return {void}
+     */
+    onIntegrationClick(): void {
+        /* no-op */
+    }
+
+    /**
+     * Gets the tooltip text for the OpenWith button
+     *
+     * @private
+     * @return {?string | ?Element}
+     */
+    getTooltip(): ?string | ?Element {
+        const { fetchError, isLoading, integrations }: State = this.state;
+        const displayIntegration = this.getDisplayIntegration();
+        if (isLoading) {
+            return null;
+        } else if (fetchError) {
```

Comment:
Using `else if` isn't necessary with guard clauses, as the prior `return` will short-circuit the remaining conditions.